### PR TITLE
Home text2/cdw42

### PR DESCRIPTION
**Pull request by mrgiles1**
## Type of change

[] New feature
<br />
[] Bug fix
<br />
[] Documentation
<br />
[] Code refactor/tech debt
<br />
[] Infrastructure

## Ticket link
[](https://linear.app/cdw/issue/CDW-42/fill-out-our-home-page)

## Part of a back end implementation?
[X] No
<br />
[] [Yes]()

### CODE REVIEW

**Comment by mrgiles1** [comment]
Not sure why my comment using the template wasn't fully incorporated... here are some screenshots on what you should see after spinning up and clicking 'MentorShip' top left:
image
image

File: frontend/src/pages/home.tsx [hunk 1]
```diff
@@ -10,39 +10,60 @@ const Home = () => {
...
+			content:
+				"As you navigate the development seas, gain experience and rise through the ranks. Both 'Captains' and 'Cadets' can unlock new features, making the journey even more rewarding.",
+			linkLabel: "Learn more... >",
+			url: "/loremipsum3"
+		},
+		{
+			headline: "Community and Support",
+			content:
+				"MentorShip is more than a platform—it's a community of passionate developers, united by a love for coding and learning. Forge lasting connections, share insights, and contribute to the growth of fellow developers.",
+			linkLabel: "Learn more... >",
```
Comment (jtabba) [inline frontend/src/pages/home.tsx]: Lets create a folder in pages for `home` and add the file to it, then create a new file called `constants.ts` and export all of these into a file. Once done rename all the variables to UPPERCASE to signify that they are unchanging variables (constants)

File: frontend/src/components/SubFeatureSection.tsx [hunk 2]
```diff
@@ -16,11 +16,15 @@ interface Props {
...
-      flexDirection='row' 
-      maxWidth='65%'
-      columnGap={8}
+      flexDirection='column' 
+      // maxWidth='90%'
+      // columnGap={8}
     >
+      {/* <Text fontSize={{ base: '2xl', md: '4xl' }} mb={4}> */}
+      <Heading fontSize={{ base: '2xl', md: '4xl' }} mb={4}>How MentorShip Works</Heading>
+      {/* </Text> */}
```
Comment (jtabba) [inline frontend/src/components/SubFeatureSection.tsx]: If not needed send it to the Netherrealm

File: frontend/src/components/MetricSection.tsx [hunk 3]
```diff
@@ -15,14 +16,30 @@ interface MetricsProps {
...
-        </Card>
+        // KEEP COMMENTED CODE BELOW IN ORDER TO REVERT TO USE TEXT FROM HOME.TSX, UPON DIRECTION FROM @JT
+        // <Card
+        //     flexDirection='row'
+        //     justifyContent='space-evenly'
+        //     p='15px'
+        //     width='65%'
+        // >
+        //     {metricsProps.map((metric, index) => <Metric key={metric.id} statNumber={metric.statNumber} statText={metric.statText}/> )}
+        // </Card>
```
Comment (jtabba) [inline frontend/src/components/MetricSection.tsx]: I said wot

**Review by jtabba — CHANGES_REQUESTED** [review]

**Pull request closed without merging**